android/guava/pom.xml

```diff
@@ -247,53 +227,4 @@
       </plugin>
     </plugins>
   </build>
-  <profiles>
-    <profile>
-      <id>srczip-parent</id>
-      <activation>
```

Comment:
All of the deletes in this PR, except for one, should disappear when #7092 is merged

Reply:
Aaand it's merged, so this has been rebased away.